fairlearn/datasets/_common.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+_DOWNLOAD_DIRECTORY_NAME = ".fairlearn-data"
```

Comment:
If you are adding this, you should add it to `_fetch_boston` too.

Reply:
[i'm not 100% sure that this constant should be in `datastes/_common.py` as opposed to one level up, but since it's not part of the exported API, we can just leave things in `datastes/_common.py` for now]

Reply:
I'm personally not a huge fan of having constants further "up" than they're needed, because it encourages people to "misuse" them for different purposes and then one ends up breaking things later or having to break things up again. You're right about _fetch_boston, of course!